samples/main.go

```diff
@@ -449,6 +449,34 @@ func main() {
 			}
 			fmt.Printf("\n\n")
 		} else if choiceN == 18 {
+			fmt.Println("You selected list VPC volumes")
+			tags := map[string]string{}
+			volName := ""
+			zoneName := ""
+			fmt.Printf("Please enter ZONE Name to filter volumes(Optional): ")
+			_, er11 = fmt.Scanf("%s", &zoneName)
+			if zoneName != "" {
+				tags["zone.name"] = zoneName
+			}
+			fmt.Printf("Please enter volume Name to filter volumes(Optional): ")
+			_, er11 = fmt.Scanf("%s", &volName)
+			if volName != "" {
+				tags["name"] = volName
+			}
+
+			start := ""
+			var limit int
+			fmt.Printf("Please enter max number of volume entries per page to be returned(Optional: ")
+			_, er11 = fmt.Scanf("%d", &limit)
+			volumeobj1, er11 := sess.ListVolumes(limit, start, tags)
```

Comment:
while printing all volumes i.e more then 50 volumes needs to be tested, I don't think so this current logic is sufficient.
also volume name I did not see a parameter which is required by RIaaS API.

Reply:
Could you please explain why current logic will break with more than 50 volumes?

Also `ListVolumes` RiaaS Api does not expect `VolumeName` field. If you want to filer particular based on volume name, then u have to add tag `name: <vol_name>`.
https://pages.github.ibm.com/riaas/api-spec/spec_scoped/#/Volumes/list_volumes

Reply:
Successfully Tested with more than 50 volumes